ingestor/src/main/scala/weco/concepts/ingestor/IngestStream.scala

```diff
@@ -1,31 +1,40 @@
 package weco.concepts.ingestor
 
-import akka.Done
+import akka.{Done, NotUsed}
 import akka.actor.ActorSystem
 import akka.http.scaladsl.Http
-import akka.stream.scaladsl.Sink
+import akka.stream.scaladsl._
 import grizzled.slf4j.Logging
-import weco.concepts.ingestor.model.IdentifierType
+import weco.concepts.ingestor.model.{Concept, IdentifierType}
 import weco.concepts.ingestor.stages.{Fetcher, Scroll, Transformer}
 
 import scala.concurrent.{ExecutionContext, Future}
 
-class IngestStream(dataUrl: String)(implicit actorSystem: ActorSystem)
-    extends Logging {
+class IngestStream(subjectsUrl: String, namesUrl: String)(implicit
```

Comment:
I wonder if this might end up as a list of urls (or map/tuple list of urls to transformers), rather than two named parameters.

Reply:
Yes, although not sure whether dissimilar sources would be better off as separate applications